idea/idea-core/src/org/jetbrains/kotlin/idea/core/overrideImplement/OverrideImplementMembersHandler.kt

```diff
@@ -106,7 +117,59 @@ abstract class OverrideImplementMembersHandler : LanguageCodeInsightActionHandle
                 selectedElements: Collection<OverrideMemberChooserObject>,
                 copyDoc: Boolean
         ) {
-            insertMembersAfter(editor, classOrObject, selectedElements.map { it.generateMember(classOrObject, copyDoc) })
+            val selectedMemberDescriptors = selectedElements.associate { it.generateMember(classOrObject, copyDoc) to it.descriptor }
+
+            val classBody = classOrObject.body
+            if (classBody == null) {
+                insertMembersAfter(editor, classOrObject, selectedMemberDescriptors.keys)
+                return
+            } else {
+                val offset = editor?.caretModel?.offset ?: classBody.startOffset
+                val offsetCursorElement = PsiTreeUtil.findFirstParent(classBody.containingFile.findElementAt(offset)) {
+                    it.parent == classBody
+                }
+                if (offsetCursorElement != null && offsetCursorElement != classBody.rBrace) {
+                    insertMembersAfter(editor, classOrObject, selectedMemberDescriptors.keys)
+                    return
+                }
+            }
+            val classLeftBrace = classBody.lBrace
+
+            val allSuperMemberDescriptors = selectedMemberDescriptors.values
```

Comment:
You should use like this:
```kotlin
val allSuperMemberDescriptors = selectedMemberDescriptors.values
    .mapNotNull { it.containingDeclaration as? ClassDescriptor }
    .associateWith {
        DescriptorUtils.getAllDescriptors(it.unsubstitutedMemberScope).filterIsInstance<CallableMemberDescriptor>()
    }
```

Reply:
Otherwise, you will only get functions that are in custom classes. For example: members from `Any` will be ignored.

Reply:
Fixed